packages/components/src/types/modal.ts

```diff
@@ -1,5 +1,3 @@
-import { EventCallback } from './callbacks';
-
 export type KoliBriModalEventCallbacks = {
-	onClose?: EventCallback<Event>;
+	onClose?: () => void;
```

Comment:
`Event` bzw. `EventCallback` benötigt immer ein Event Parameter, das wir hier nicht haben, wenn das Modal von "außen" geschlossen wurde.

Reply:
Man könnte ein eigenes Event erzeugen. So wird es z. B. in der Card gemacht:

```ts
this._on.onClose(new Event('Close'));
```